domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/gtfs/fareattributes/FareAttribute.java

```diff
@@ -0,0 +1,158 @@
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public class FareAttribute {
+
+    @NotNull
+    final String fareId;
+
+    private final float price;
+
+    @NotNull
+    final String currencyType;
+
+    @NotNull
+    final PaymentMethod paymentMethod;
+
+    @NotNull
+    final Transfers transfers;
+
+    @Nullable
+    private final String agencyId;
+    @Nullable
+    private final Integer transferDuration;
+
+    @NotNull
+    public String getFareId() {
+        return fareId;
+    }
+
+    public float getPrice() {
+        return price;
+    }
+
+    @NotNull
+    public String getCurrencyType() {
+        return currencyType;
+    }
+
+    @NotNull
+    public PaymentMethod getPaymentMethod() {
+        return paymentMethod;
+    }
+
+    @NotNull
+    public Transfers getTransfers() {
+        return transfers;
+    }
+
+    @Nullable
+    public String getAgencyId() {
+        return agencyId;
+    }
+
+    @Nullable
+    public Integer getTransferDuration() {
+        return transferDuration;
+    }
+
+    public FareAttribute(@NotNull final String fareId,
+                         final float price,
+                         @NotNull final String currencyType,
+                         @NotNull final PaymentMethod paymentMethod,
+                         @NotNull final Transfers transfers,
+                         @Nullable final String agencyId,
+                         @Nullable final Integer transferDuration) {
+        this.fareId = fareId;
+        this.price = price;
+        this.currencyType = currencyType;
+        this.paymentMethod = paymentMethod;
+        this.transfers = transfers;
+        this.agencyId = agencyId;
+        this.transferDuration = transferDuration;
+    }
+
+    public static class FareAttributeBuilder {
+        private String fareId;
+        private Float price;
+        private String currencyType;
+        private PaymentMethod paymentMethod;
+        private Transfers transfers;
+        private String agencyId;
+        private Integer transferDuration;
+
+        public FareAttributeBuilder fareId(String fareId) {
+            this.fareId = fareId;
+            return this;
+        }
+
+        public FareAttributeBuilder price(Float price) {
+            this.price = price;
+            return this;
+        }
+
+        public FareAttributeBuilder currencyType(String currencyType) {
+            this.currencyType = currencyType;
+            return this;
+        }
+
+        public FareAttributeBuilder paymentMethod(Integer paymentMethod) {
+            this.paymentMethod = PaymentMethod.fromInt(paymentMethod);
+            return this;
+        }
+
+        public FareAttributeBuilder transfers(Integer transfers) {
+            this.transfers = Transfers.fromInt(transfers);
+            return this;
+        }
+
+        public FareAttributeBuilder agencyId(@Nullable String agencyId) {
+            this.agencyId = agencyId;
```

Comment:
`agency_id` is conditionally required here like in the other files, if agency.txt contains multiple records. I'd suggest adding a `// TODO` statement here pointing to the issue that we have open for this so it's easier to track down later.